modules/view-system-access-scopes.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-role-based-access-control.adoc
+:_module-type: PROCEDURE
+[id="view-system-access-scopes_{context}"]
+= Viewing the details for a system access scope
+
+You can view the allowed and not allowed Kubernetes and {ocp} resources for an access scope in the RHACS portal.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Access control*.
+. Select *Access scopes*.
+. Click on one of the access scope to view its details. The details page shows a list of clusters and namespaces, and which of them are allowed for the selected access scope.
```

Comment:
- s/access scope/access scopes
- It's a bit confusing what "them" is referring to in "which of them are allowed", so maybe you could say something like, "and which ones are allowed"?